.github/scripts/build-size.js

```diff
@@ -308,9 +332,33 @@ if (require.main === module) {
       const decimals = options["d"] || options["decimals"] || FLAG_INFO["decimals"].default;
       const binary = options["b"] || options["binary"] || FLAG_INFO["binary"].default;
       const outfile = options["o"] || options["outfile"]; // no default
+      const runtime = options["r"] || options["runtime"]; // no default
 
       const buildSizes = await getBuildSizes(path, type);
 
+      if(runtime) {
+        const performanceInfo = await getPerformanceInfo(path, buildSizes.mainBundleName);
+
+        // log sizes to console
+        const title = "|> Application Performance <|";
+        const line = "-".repeat(title.length);
+        const bundle = `Main ${type.toUpperCase()} bundle`;

```

Comment:
One more thing, you could probably delete the line/bundle variables here, and move the ones below up above the runtime conditional. Or move the runtime conditional below the build sizes logging

Reply:
Ah, good catch.